README.md

```diff
@@ -1,42 +1,49 @@
+[repositoryUrl]: https://github.com/actualbudget
+[repositoryDocs]: https://github.com/actualbudget/actual/tree/master/docs
+[repositoryRelease]: https://github.com/actualbudget/actual/blob/master/docs/releasing.md
+[repositoryContributing]: https://github.com/actualbudget/actual/blob/master/CONTRIBUTING.md
 
 This is the source code for [Actual](https://actualbudget.com), a local-first personal finance tool. It is 100% free and open-source.
 
 If you are only interested in running the latest version, you don't need this repo. You can get the latest version through npm.
 
-More docs are available in the [docs](https://github.com/actualbudget/actual/tree/master/docs) folder.
+More docs are available in the [docs][repositoryDocs] folder.
 
-If you are interested in contributing, or want to know how development works, see [CONTRIBUTING.md](https://github.com/actualbudget/actual/blob/master/CONTRIBUTING.md)
+If you are interested in contributing, or want to know how development works, see [CONTRIBUTING.md][repositoryContributing]
 
 Join the [discord](https://discord.gg/pRYNYr4W5A)!
 
 ## Installation
 
 ### The easy way: using a server (recommended)
 
-The easiest way to get Actual running is to use the [actual-server](https://github.com/actualbudget/actual-server) project. That is the server for syncing changes across devices, and it comes with the latest version of Actual. The server will provide both the web project and a server for syncing.
+The easiest way to get Actual running is to use the [actual-server][repositoryUrl] project. That is the server for syncing changes across devices, and it comes with the latest version of Actual. The server will provide both the web project and a server for syncing.
+- This project use Yarn as main package manager as *yarn.lock* file is provided but feel free to delete the file and use NPM, PNPM, etc... and use proper commands.
```

Comment:
```suggestion
- This project use Yarn as its main package manager, other package managers may be used but stability can't be guaranteed.
```

I think this could cause some headaches down the line. The setup of this repo is quite fragile and I don't think we should give the impression that we support all package managers.

Reply:
I totally understand Your opinion and I've agree, let's change it this way, thank You :)